lib/phoenix/endpoint/cowboy_handler.ex

```diff
@@ -112,6 +112,6 @@ defmodule Phoenix.Endpoint.CowboyHandler do
 
   defp info(scheme, endpoint, ref) do
     port = :ranch.get_port(ref)
-    "Running #{inspect endpoint} with Cowboy using #{scheme} on port #{port}"
+    "Running #{inspect endpoint} with Cowboy using #{scheme} on http://localhost:#{port}"
```

Comment:
Can you then please interpolate the scheme too? Something like:

    Running #{inspect endpoint} with Cowboy using #{scheme}://localhost:#{port}

Thank you!